src/models/generated/ThermalMultiStart.jl

```diff
@@ -0,0 +1,237 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+"""
+    mutable struct ThermalMultiStart <: ThermalGen
+        name::String
+        available::Bool
+        status::Bool
+        bus::Bus
+        activepower::Float64
+        reactivepower::Float64
+        rating::Float64
+        primemover::PrimeMovers.PrimeMover
+        fuel::ThermalFuels.ThermalFuel
+        activepowerlimits::Min_Max
+        reactivepowerlimits::Union{Nothing, Min_Max}
+        ramplimits::Union{Nothing, NamedTuple{(:up, :down), Tuple{Float64, Float64}}}
+        power_trajectory::Union{Nothing, NamedTuple{(:startup, :shutdown), Tuple{Float64, Float64}}}
```

Comment:
Can you please add a comment about what this field is?

Reply:
will do